### PR TITLE
Transfer "FOLLOW US" section in the footer

### PR DESCRIPTION
**Pull request by yungboigrek**

### CODE REVIEW

File: src/main/java/common/BasePage.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 public class BasePage {
     public static final Logger logger = LogManager.getLogger(BasePage.class);
     private final int BASE_WAIT = 5;
-    protected static WebDriver driver;
+    public static WebDriver driver;
```
Comment (Goldbe54) [inline src/main/java/common/BasePage.java:19]: Why these changes?

File: src/main/java/common/CommonActions.java
Comment (Goldbe54) [inline src/main/java/common/CommonActions.java:1]: Why not to use SneakyThrows

File: src/main/java/fragments/FooterFragment.java [hunk 2]
```diff
@@ -15,6 +15,12 @@ public class FooterFragment extends BasePage {
     private static final String ACCEPT_PRIVACY_POLICY_CHECKBOX = id("newsletterCheckField");
     private static final String PRIVACY_POLICY_TEXT = "//*[@class='text-wrapper']";
     private static final String PRIVACY_POLICY_LINK = PRIVACY_POLICY_TEXT + "/a";
+    private static final String TWITTER_ICON = "//a[@href='https://twitter.com/vans_europe']";
+    private static final String FACEBOOK_ICON = "//a[@href='https://www.facebook.com/VansEurope']";
+    private static final String INSTAGRAM_ICON = "//a[@href='https://www.instagram.com/vans_europe/']";
+    private static final String YOUTUBE_ICON = "//a[@href='https://www.youtube.com/user/VansShoes66']";
+    private static final String FOLLOW_US = "//div[@id='socialLinksFooterBox']";
+
```
Comment (Goldbe54) [inline src/main/java/fragments/FooterFragment.java:23]: Use `.id()` method, and delete empty line

File: src/main/java/fragments/FooterFragment.java [hunk 3]
```diff
@@ -51,4 +57,26 @@ public WebElement getAcceptPrivacyPolicyText() {
...
+    {
+        waitClickabilityOfElement(INSTAGRAM_ICON).click();
+    }
+
+    public void goToYoutubeChannel()
+    {
+        waitClickabilityOfElement(YOUTUBE_ICON).click();
+    }
+
+    public WebElement getFollowUsSection(){return waitPresenceOfElement(FOLLOW_US);}
```
Comment (Goldbe54) [inline src/main/java/fragments/FooterFragment.java:81]: Use the generally accepted rules for declaring methods

File: src/main/java/pages/Homepage.java
Comment (Goldbe54) [inline src/main/java/pages/Homepage.java:1]: Delete unused imports

File: src/test/java/listeners/TestListener.java
Comment (Goldbe54) [inline src/test/java/listeners/TestListener.java:1]: What are the reasons for these changes?

File: src/test/java/tests/BaseTest.java
Comment (Goldbe54) [inline src/test/java/tests/BaseTest.java:1]: Pls, remove all changes

File: src/test/java/tests/CheckFollowUsButtonTest.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    private static final String EXPECTED_FACEBOOK_URL = "https://www.facebook.com/VansEurope";
+    private static final String EXPECTED_INSTAGRAM_URL = "https://www.instagram.com/vans_europe/";
+    private static final String EXPECTED_PART_YOUTUBE_URL = "https://www.youtube.com/user/VansShoes66";
+
+    private static FooterFragment footerFragment;
+
+    @BeforeMethod
+    public void initTest() {
+        footerFragment = new FooterFragment(driver);
+
```
Comment (Goldbe54) [inline src/test/java/tests/CheckFollowUsButtonTest.java:20]: delete empty line

File: src/test/java/tests/CheckFollowUsButtonTest.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    }
+
+    @Test(description = "SPRIN-114 Transfer FOLLOW US section in the footer")
+    public void clickOnTheTwitterButton() {
```
Comment (Goldbe54) [inline src/test/java/tests/CheckFollowUsButtonTest.java:24]: rename method. eg: validateFollowUsSection

File: src/test/java/tests/CheckFollowUsButtonTest.java [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        scrollToElement(driver,footerFragment.getFollowUsSection());
+        footerFragment.goToTwitterPage();
+        driver.getCurrentUrl();
```
Comment (Goldbe54) [inline src/test/java/tests/CheckFollowUsButtonTest.java:27]: delete unuseed instructions

File: src/test/java/tests/CheckFollowUsButtonTest.java [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        footerFragment.goToInstagramPage();
+
+        softAssert.assertEquals(driver.getCurrentUrl(),EXPECTED_INSTAGRAM_URL,"Expected URL isn`t valid");
+
+        driver.navigate().back();
+        footerFragment.goToYoutubeChannel();
+
+        softAssert.assertTrue(driver.getCurrentUrl().startsWith(EXPECTED_PART_YOUTUBE_URL),"Expected URL isn`t valid");
+    }
+
```
Comment (Goldbe54) [inline src/test/java/tests/CheckFollowUsButtonTest.java:46]: delete empty line

File: src/test/java/tests/CheckFollowUsButtonTest.java [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+}
+
```
Comment (Goldbe54) [inline src/test/java/tests/CheckFollowUsButtonTest.java:48]: ditto

**Review by Goldbe54 — APPROVED** [review]

**Review by KhraschukZahar — APPROVED** [review]
